core/statistics/machine/netStatistics.go

```diff
@@ -119,3 +138,13 @@ func (ns *NetStatistics) PercentSent() uint64 {
 func (ns *NetStatistics) PercentRecv() uint64 {
 	return atomic.LoadUint64(&ns.percentRecv)
 }
+
+// TotalBytesSendInCurrentEpoch returns total bytes send in current epoch
```

Comment:
// TotalBytesSendInCurrentEpoch returns the number of bytes sent in the current epoch *
also rename func name to `Sent` instead of `Send`

Reply:
done.